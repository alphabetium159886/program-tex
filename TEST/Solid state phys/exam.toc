\contentsline {section}{\numberline {1}什么是声子}{4}{section.1}%
\contentsline {section}{\numberline {2}什么是Bloch电子}{4}{section.2}%
\contentsline {subsection}{\numberline {2.1}Bloch振荡能被观测到吗？}{5}{subsection.2.1}%
\contentsline {section}{\numberline {3}能帶寬度}{5}{section.3}%
\contentsline {subsection}{\numberline {3.1}怎么算}{6}{subsection.3.1}%
\contentsline {section}{\numberline {4}导带和价带}{6}{section.4}%
\contentsline {section}{\numberline {5}紧束缚近似}{7}{section.5}%
\contentsline {subsection}{\numberline {5.1}怎么计算}{7}{subsection.5.1}%
\contentsline {section}{\numberline {6}一个非常干净的材料是超导体吗}{8}{section.6}%
\contentsline {section}{\numberline {7}De Haas－Van Alphen 效应}{8}{section.7}%
\contentsline {section}{\numberline {8}朗道能级}{9}{section.8}%
\contentsline {subsection}{\numberline {8.1}朗道能级如何计算}{10}{subsection.8.1}%
\contentsline {section}{\numberline {9}格波的声学支和光学支}{10}{section.9}%
\contentsline {section}{\numberline {10}3-dimension}{11}{section.10}%
\contentsline {section}{\numberline {11}density of state}{11}{section.11}%
\contentsline {section}{\numberline {12}玻恩－卡门周期性边界条件}{12}{section.12}%
\contentsline {section}{\numberline {13}什么是声子，并且请将声子的性质与光子用表格做比较}{12}{section.13}%
\contentsline {section}{\numberline {14}Du Long-Petit 定律是什么}{13}{section.14}%
\contentsline {section}{\numberline {15}一定温度下，是光学模式的声子数目多，还是声学模式的声子数目多}{13}{section.15}%
\contentsline {section}{\numberline {16}同一震动模式下又是哪个模式（声or光）的声子多}{13}{section.16}%
\contentsline {section}{\numberline {17}同一震动模式下，温度低的声子多还是温度高的声子多}{14}{section.17}%
\contentsline {section}{\numberline {18}声子数目是否守恒？ 高温时，频率为$\omega $的格波声子数目与温度有何关系}{14}{section.18}%
\contentsline {section}{\numberline {19}晶体在绝对零度时，还有声子（或格波）存在吗？}{15}{section.19}%
\contentsline {section}{\numberline {20}请简述Debye模型}{15}{section.20}%
\contentsline {section}{\numberline {21}请简述爱因斯坦的固体热容模型}{16}{section.21}%
\contentsline {section}{\numberline {22}请用能带理论解释导体与半导体与非导体}{16}{section.22}%
\contentsline {section}{\numberline {23}解释空穴}{17}{section.23}%
\contentsline {section}{\numberline {24}声子对材料的比热有什么贡献}{18}{section.24}%
\contentsline {section}{\numberline {25}debye温度的物理意义}{18}{section.25}%
\contentsline {section}{\numberline {26}分别利用爱因斯坦模型与Debye模型计算声子对比热的贡献}{19}{section.26}%
\contentsline {section}{\numberline {27}在半经典近似下，写出周期性势场中运动的电子在外电场作用下的运动方程，并以此为基础解释bloch振荡}{19}{section.27}%
\contentsline {section}{\numberline {28}写出磁场下电子的半经典运动方程}{20}{section.28}%
\contentsline {section}{\numberline {29}请用量子力学解释共价键的形成原因}{20}{section.29}%
\contentsline {section}{\numberline {30}紧束缚近似下，简单立方晶体由原子$p_x$, $p_y$, $p_z$态形成的能带与s态形成的能带有什么不同}{21}{section.30}%
\contentsline {section}{\numberline {31}解释霍尔效应与量子霍尔效应}{22}{section.31}%
\contentsline {section}{\numberline {32}描述Drude模型的物理图像}{23}{section.32}%
