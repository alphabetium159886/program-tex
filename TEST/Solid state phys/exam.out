\BOOKMARK [1][-]{section.1}{\376\377\116\300\116\110\146\057\130\360\133\120}{}% 1
\BOOKMARK [1][-]{section.2}{\376\377\116\300\116\110\146\057\000B\000l\000o\000c\000h\165\065\133\120}{}% 2
\BOOKMARK [2][-]{subsection.2.1}{\376\377\000B\000l\000o\000c\000h\143\057\203\141\200\375\210\253\211\302\155\113\122\060\124\027\377\037}{section.2}% 3
\BOOKMARK [1][-]{section.3}{\376\377\200\375\136\066\133\354\136\246}{}% 4
\BOOKMARK [2][-]{subsection.3.1}{\376\377\140\016\116\110\173\227}{section.3}% 5
\BOOKMARK [1][-]{section.4}{\376\377\133\374\136\046\124\214\116\367\136\046}{}% 6
\BOOKMARK [1][-]{section.5}{\376\377\175\047\147\137\177\032\217\321\117\074}{}% 7
\BOOKMARK [2][-]{subsection.5.1}{\376\377\140\016\116\110\213\241\173\227}{section.5}% 8
\BOOKMARK [1][-]{section.6}{\376\377\116\000\116\052\227\136\136\070\136\162\121\300\166\204\147\120\145\231\146\057\215\205\133\374\117\123\124\027}{}% 9
\BOOKMARK [1][-]{section.7}{\376\377\000D\000e\000\040\000H\000a\000a\000s\377\015\000V\000a\000n\000\040\000A\000l\000p\000h\000e\000n\000\040\145\110\136\224}{}% 10
\BOOKMARK [1][-]{section.8}{\376\377\147\027\220\123\200\375\176\247}{}% 11
\BOOKMARK [2][-]{subsection.8.1}{\376\377\147\027\220\123\200\375\176\247\131\202\117\125\213\241\173\227}{section.8}% 12
\BOOKMARK [1][-]{section.9}{\376\377\150\074\154\342\166\204\130\360\133\146\145\057\124\214\121\111\133\146\145\057}{}% 13
\BOOKMARK [1][-]{section.10}{\376\377\0003\000-\000d\000i\000m\000e\000n\000s\000i\000o\000n}{}% 14
\BOOKMARK [1][-]{section.11}{\376\377\000d\000e\000n\000s\000i\000t\000y\000\040\000o\000f\000\040\000s\000t\000a\000t\000e}{}% 15
\BOOKMARK [1][-]{section.12}{\376\377\163\273\140\151\377\015\123\141\225\350\124\150\147\037\140\047\217\271\165\114\147\141\116\366}{}% 16
\BOOKMARK [1][-]{section.13}{\376\377\116\300\116\110\146\057\130\360\133\120\377\014\136\166\116\024\213\367\134\006\130\360\133\120\166\204\140\047\215\050\116\016\121\111\133\120\165\050\210\150\150\074\120\132\153\324\217\203}{}% 17
\BOOKMARK [1][-]{section.14}{\376\377\000D\000u\000\040\000L\000o\000n\000g\000-\000P\000e\000t\000i\000t\000\040\133\232\137\213\146\057\116\300\116\110}{}% 18
\BOOKMARK [1][-]{section.15}{\376\377\116\000\133\232\156\051\136\246\116\013\377\014\146\057\121\111\133\146\152\041\137\017\166\204\130\360\133\120\145\160\166\356\131\032\377\014\217\330\146\057\130\360\133\146\152\041\137\017\166\204\130\360\133\120\145\160\166\356\131\032}{}% 19
\BOOKMARK [1][-]{section.16}{\376\377\124\014\116\000\227\007\122\250\152\041\137\017\116\013\123\310\146\057\124\352\116\052\152\041\137\017\377\010\130\360\000o\000r\121\111\377\011\166\204\130\360\133\120\131\032}{}% 20
\BOOKMARK [1][-]{section.17}{\376\377\124\014\116\000\227\007\122\250\152\041\137\017\116\013\377\014\156\051\136\246\117\116\166\204\130\360\133\120\131\032\217\330\146\057\156\051\136\246\232\330\166\204\130\360\133\120\131\032}{}% 21
\BOOKMARK [1][-]{section.18}{\376\377\130\360\133\120\145\160\166\356\146\057\124\046\133\210\140\122\377\037\000\040\232\330\156\051\145\366\377\014\230\221\163\207\116\072\166\204\150\074\154\342\130\360\133\120\145\160\166\356\116\016\156\051\136\246\147\011\117\125\121\163\174\373}{}% 22
\BOOKMARK [1][-]{section.19}{\376\377\146\166\117\123\127\050\176\335\133\371\226\366\136\246\145\366\377\014\217\330\147\011\130\360\133\120\377\010\142\026\150\074\154\342\377\011\133\130\127\050\124\027\377\037}{}% 23
\BOOKMARK [1][-]{section.20}{\376\377\213\367\173\200\217\360\000D\000e\000b\000y\000e\152\041\127\213}{}% 24
\BOOKMARK [1][-]{section.21}{\376\377\213\367\173\200\217\360\162\061\126\340\145\257\127\146\166\204\126\372\117\123\160\355\133\271\152\041\127\213}{}% 25
\BOOKMARK [1][-]{section.22}{\376\377\213\367\165\050\200\375\136\046\164\006\213\272\211\343\221\312\133\374\117\123\116\016\123\112\133\374\117\123\116\016\227\136\133\374\117\123}{}% 26
\BOOKMARK [1][-]{section.23}{\376\377\211\343\221\312\172\172\172\164}{}% 27
\BOOKMARK [1][-]{section.24}{\376\377\130\360\133\120\133\371\147\120\145\231\166\204\153\324\160\355\147\011\116\300\116\110\215\041\163\056}{}% 28
\BOOKMARK [1][-]{section.25}{\376\377\000d\000e\000b\000y\000e\156\051\136\246\166\204\162\151\164\006\141\017\116\111}{}% 29
\BOOKMARK [1][-]{section.26}{\376\377\122\006\122\053\122\051\165\050\162\061\126\340\145\257\127\146\152\041\127\213\116\016\000D\000e\000b\000y\000e\152\041\127\213\213\241\173\227\130\360\133\120\133\371\153\324\160\355\166\204\215\041\163\056}{}% 30
\BOOKMARK [1][-]{section.27}{\376\377\127\050\123\112\176\317\121\170\217\321\117\074\116\013\377\014\121\231\121\372\124\150\147\037\140\047\122\277\127\072\116\055\217\320\122\250\166\204\165\065\133\120\127\050\131\026\165\065\127\072\117\134\165\050\116\013\166\204\217\320\122\250\145\271\172\013\377\014\136\166\116\345\153\144\116\072\127\372\170\100\211\343\221\312\000b\000l\000o\000c\000h\143\057\203\141}{}% 31
\BOOKMARK [1][-]{section.28}{\376\377\121\231\121\372\170\301\127\072\116\013\165\065\133\120\166\204\123\112\176\317\121\170\217\320\122\250\145\271\172\013}{}% 32
\BOOKMARK [1][-]{section.29}{\376\377\213\367\165\050\221\317\133\120\122\233\133\146\211\343\221\312\121\161\116\367\225\056\166\204\137\142\142\020\123\237\126\340}{}% 33
\BOOKMARK [1][-]{section.30}{\376\377\175\047\147\137\177\032\217\321\117\074\116\013\377\014\173\200\123\125\172\313\145\271\146\166\117\123\165\061\123\237\133\120\000p\000x\000,\000\040\000p\000y\000,\000\040\000p\000z\140\001\137\142\142\020\166\204\200\375\136\046\116\016\000s\140\001\137\142\142\020\166\204\200\375\136\046\147\011\116\300\116\110\116\015\124\014}{}% 34
\BOOKMARK [1][-]{section.31}{\376\377\211\343\221\312\227\015\134\024\145\110\136\224\116\016\221\317\133\120\227\015\134\024\145\110\136\224}{}% 35
\BOOKMARK [1][-]{section.32}{\376\377\143\317\217\360\000D\000r\000u\000d\000e\152\041\127\213\166\204\162\151\164\006\126\376\120\317}{}% 36
